sos/report/plugins/auditd.py

```diff
@@ -33,9 +33,20 @@ def setup(self):
             "auditctl -l"
         ])
 
+        config_file = "/etc/audit/auditd.conf"
+        log_file = "/var/log/audit/audit.log"
+        try:
+            for line in open(config_file).read().splitlines():
```

Comment:
This can leave an open file handle since there is no context manager or an explicit close. Using a context manager like below will ensure it gets closed once we're out of this loop.

```python
with open(config_file, 'r') as cf:
    for line in cf.read().splitlines()
        [...]
```

Reply:
And the 'with open...' still needs to happen inside the  try section, to check if the file exist, right?

Reply:
Correct